tests/test_fyle/test_task.py

```
@@ -57,7 +53,7 @@ def test_create_expense_groups(mocker, db):
     task_log = TaskLog.objects.get(id=task_log.id)
     assert task_log.status == "FAILED"
 
-    with mock.patch("fyle.platform.apis.v1beta.admin.Expenses.list_all") as mock_call:
+    with mock.patch("fyle.platform.apis.v1.admin.Expenses.list_all") as mock_call:
```

Ruff (0.8.2)

Redefinition of unused `mock_call` from line 21

(F811)